build-system/tasks/changelog.js

```diff
@@ -38,8 +38,7 @@ const {GITHUB_ACCESS_TOKEN} = process.env;
 const exec = BBPromise.promisify(childProcess.exec);
 const gitExec = BBPromise.promisify(git.exec);
```

Comment:
Cleanup task for another PR: If it makes sense, we can consolidate this code with the code in [build-system/git.js](https://github.com/ampproject/amphtml/blob/master/build-system/git.js).

Reply:
sounds good

Reply:
filed here https://github.com/ampproject/amphtml/issues/20138